.github/workflows/rust.yml

```diff
@@ -119,10 +119,6 @@ jobs:
           - x86_64-unknown-illumos
           - x86_64-unknown-netbsd
           - x86_64-linux-android
-          - i686-linux-android
-          - arm-linux-androideabi
-          - aarch64-linux-android
-          - sparcv9-sun-solaris
```

Comment:
Why drop solaris? It seems to me like we should keep that.

Reply:
You're right. I don't know why I dropped the solaris line. :)